app/enterprise/1.5.x/kong-for-kubernetes/deployment-options.md

```diff
@@ -0,0 +1,179 @@
+These deployments use the `kong-enterprise-k8s` image and must set
+`KONG_DATABASE=off` in their environment variables.
+
+Database-backed deployments offer a wider range of features using the
+`kong-enterprise-edition` image. Review the sections below to determine if your
+use case requires a feature that is not available in DB-less deployments.
+
+## Feature availability
+
+Some Kong Enterprise features are not available in DB-less deployments. Users
+should use the `kong-enterprise-edition` image and a database-backed deployment
+if they wish to use:
+
+* Kong Manager
+* Dev Portal
+* Brain
+* Immunity
+* Teams (RBAC)
+* Vitals
+* Workspaces
+
+Because Kong for Kubernetes is configured by the ingress controller, some
+functionality in these features is different from traditional deployments:
+
+* Users will not typically create proxy configuration through Kong Manager;
+  proxy configuration is normally managed by the controller and users provide
+  configuration via Kubernetes resources.
+* Because the controller creates proxy configuration on behalf of users, users
+  do not need to interact with the Admin API directly. Typical Kong for
+  Kubernetes deployments do not expose the Admin API to protect it in lieu of
+  RBAC: only the controller can access it. Kubernetes-level RBAC rules and
+  namespaces should be used to restrict what configuration administrators can
+  create.
+* Ingress controller instances create configuration in a single workspace only
+  (`default` by default). To use multiple workspaces, users should deploy
+  multiple controller instances, setting the `CONTROLLER_KONG_WORKSPACE`
+  environment variable to the workspace that instance should use. These
+  instances should set `CONTROLLER_INGRESS_CLASS` to unique values, to avoid
+  creating duplicate configuration in workspaces. Note that if controller
+  instances are deployed outside the Kong pod the Admin API must be exposed,
+  and users should enable RBAC with workspace admin users for the controllers.
+  Set `CONTROLLER_KONG_ADMIN_TOKEN` to the RBAC user's token.
+* The controller cannot manage configuration for the features above: it cannot
+  create workspaces, Dev Portal content, admins, etc. These features must be
+  configured manually through the Admin API.
+
+## Plugin compatibility
+
+Not all plugins are compatible with DB-less operation, and as such not all
+plugins are available in the `kong-enterprise-k8s` image. In addition to the
+[compatible core plugins][core-plugins], the `kong-enterprise-k8s` image
+includes the following Enterprise plugins:
+
+* canary release
+* collector
+* degraphql
+* forward-proxy
+* graphql-proxy-cache-advanced
+* graphql-rate-limiting-advanced
+* jwt-signer
+* kafka-log
+* kafka-upstream
+* ldap-auth-advanced
+* mtls-auth
+* oauth2-introspection
+* openid-connect
+* proxy-cache-advanced
+* rate-limiting-advanced
+* request-validator
+* response-transformer-advanced
+* vault-auth
```

Comment:
Not in scope for this PR, but something to look into: for all of the plugins in this list, is it safe to add a section in the docs on db-less config as mentioned in this ticket: https://konghq.atlassian.net/browse/FTI-1462

Reply:
Strictly speaking yes, though with the caveat that any plugin can use that format--as I understand it the main ask there is to provide the YAML version for decK rather than DB-less.